NicolaeChristof/Rise
# Feature credits and divider

This branch adds the credits to the game. Their accessible from the main menu under the "Credits" option, and the credits scroll upward and loop once they're finished. Additionally, I added the divider that's activated when the "Two Players" option.

How to Test:
Make sure that the credits loop correctly and they play at a reasonable speech. Also make sure that the divider only comes up when the game is being played and is in multiplayer.
